front/src/modules/trainschedule/components/ManageTrainSchedule/Map.tsx

```diff
@@ -44,13 +42,15 @@ import IGN_CADASTRE from 'common/Map/Layers/IGN_CADASTRE';
 import { CUSTOM_ATTRIBUTION } from 'common/Map/const';
 import LineSearchLayer from 'common/Map/Layers/LineSearchLayer';
 import Terrain from 'common/Map/Layers/Terrain';
-import { getTerrain3DExaggeration } from 'reducers/map/selectors';
+import { getMap, getTerrain3DExaggeration } from 'reducers/map/selectors';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
 import { getMapMouseEventNearestFeature } from '../../../../utils/mapHelper';
 
 function Map() {
-  const { viewport, mapSearchMarker, mapStyle, showOSM, layersSettings } = useSelector(
-    (state: RootState) => state.map
-  );
+  const { viewport, mapSearchMarker, mapStyle, showOSM, layersSettings } = useSelector(getMap);
```

Comment:
Could you group the `useSelectors` together :)

Reply:
Done